src/cascade/executor/argument_parser.py

```diff
@@ -99,70 +103,135 @@ def exit(self, status=0, message=None):
         raise ArgumentException(message, status)
 
     @staticmethod
-    def _logging_config(args, epiviz_log_dir=EPIVIZ_LOG_DIR, code_log_dir=CODE_LOG_DIR):
+    def _logging_config(args):
         """Configures logging. Command-line arguments ``-v`` and ``-q``
         set both the code log file and what streams to stdout. The ``-v``
         flag turns on debug level, and ``-q`` sets it to info level.
         The math logger will always be set to info level.
```

Comment:
Isn't the math logger always set to the debug level?

Reply:
Yes, that comment is wrong. Nice catch. Really appreciate it.